woudc_data_registry/util.py

```diff
@@ -53,6 +53,8 @@
 import uuid
 
 
+import base64
```

Comment:
is `base64` needed here if we're calculating the sha512 hash with `hashlib` only?